ecosystem/sep-0038.md

```diff
@@ -208,3 +210,26 @@ Name | Type | Description
 `price` | string | The price offered by the anchor for one unit of `buy_asset` in terms of `sell_asset`. In traditional finance, `buy_asset` would be referred to as the base asset and `sell_asset` as the counter asset.
 `buy_amount` | string | The amount of `buy_asset` to be exchanged for `sell_asset`. `price * sell_amount = buy_amount` must be true up to the number of decimals required for `buy_asset`.
 `sell_amount` | string | The amount of `sell_asset` to be exchanged for `buy_asset`.
+
+### GET Quote
+
+This endpoint can be used to fetch a previously-provided firm quote. Expired quotes should still be retrievable using this endpoint.
```

Comment:
>Expired quotes should still be retrievable using this endpoint.

❓ What's the motivation for this? It is convenient for clients but at a cost for servers. I think quotes should be transient so that there is not undue stress on servers to maintain large amounts of historical data.

Reply:
The primary motivation is to keep record of quotes used by other protocols, but we could allow unused quotes to be removed at expiration.